docs/pages/desktop-access/getting-started.mdx

```diff
@@ -27,104 +25,133 @@ This guide requires you to have:
+### Deny interactive login
 
-<Figure align="left" bordered caption="Remove Authenticated">
-  ![Remove Authenticated](../../img/desktop-access/remove-authenticated.png)
-</Figure>
+1. From the `Group Policy Management` window, right-click on the GPO you just created (`Block teleport-svc Interactive Login`) and select `Edit...` from the context menu.
+2. Select:
 
-The "Remove" step prevents the GPO from being applied to all authenticated users in this domain, since we only want it to apply to members 
-of the security Group created above (`svc-teleport-group`), of which only the User created above is a member (`svc-teleport`). 
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > Local Policies > User Rights Assignment
+```
 
-To make that the case, in the same "Security Filtering" pane select "Add...", and enter the name of the Group (`svc-teleport-group`)
-in the resultant popup, clicking "Check Names" to confirm the spelling before pressing "OK". After this step, only the `svc-teleport-group`
-should be displayed in the "Security Filtering" pane:
+3. Double click `Deny log on locally` and in the popup, check `Define these policy settings`.
+4. Then click `Add User or Group...`, `Browse ...`, enter the SAM account name of the user you created above (`svc-teleport`) and hit `Check Names` select your Group, and then hit `OK` on all the windows.
+5. Repeat steps 3 and 4 for `Deny log on through Remote Desktop Services` (in lieu of `Deny log on locally`).
 
-<Figure align="left" bordered caption="Apply GPO">
-  ![Apply GPO](../../img/desktop-access/apply-gpo.png)
+<Figure align="left" bordered caption="Deny Interactive Login">
+  ![Deny Interactive Login](../../img/desktop-access/deny-interactive-login.png)
 </Figure>
 
-## Step 2/6. Configure Group Policy to allow Teleport connections
+## Step 3/6: Configure a GPO to allow Teleport connections
 
-{/* TODO: script this using PowerShell */}
+Next, we need to configure a GPO to allow Teleport desktop sessions. This includes telling your computers to trust Teleport's CA, allowing the certificate-based smart card authentication, and ensuring RDP is enabled.
 
-Next, we need to configure our Group Policy Object to trust Teleport for user
-authentication and allow the certificate-based mechanism that Teleport uses under the hood
-(smart cards).
+### Export the Teleport CA
 
-<Admonition
-  type="note"
-  title="Requires Existing Cluster"
->
-  The following step requires an existing cluster. If you don't already have a Teleport cluster up and running,
-  see our general [Getting Started](https://goteleport.com/docs/getting-started/) guide.
+<Admonition type="note" title="Requires Existing Cluster">
+  The following step requires an existing cluster. If you don't already have a
+  Teleport cluster up and running, see our general [Getting
+  Started](https://goteleport.com/docs/getting-started/) guide.
```

Comment:
Please use local file notation instead of global URL notation when referring another article `[Example](../path/to/file.mdx)`